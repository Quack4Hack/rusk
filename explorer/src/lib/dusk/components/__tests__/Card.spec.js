import { afterEach, describe, expect, it } from "vitest";
import { cleanup, render } from "@testing-library/svelte";
import { Card } from "..";

describe("Card", () => {
  afterEach(cleanup);

  it("renders the Card component", () => {
    const { container } = render(Card);

    expect(container.firstChild).toMatchSnapshot();
  });
});
