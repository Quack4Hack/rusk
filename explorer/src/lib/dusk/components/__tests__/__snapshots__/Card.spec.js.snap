// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card > renders the Card component 1`] = `
<div>
  <div
    class="dusk-card"
  >
     
     
  </div>
  
</div>
`;
